### DISCUSSION TITLE
Documentation for NestedView & NestedElementBase?

### DISCUSSION THREAD

**Discussion by iamntz** [elementor/elementor#24430] [Developers Community]
It seems there are a couple of widgets that can have nested elements (Accordion, Nested Carousel and some others).

Digging through code, I found `NestedView`, `NestedElementBase`, etc classes that makes possible to have the editor displaying the containers.

https://github.com/elementor/elementor/tree/main/modules/nested-accordion/assets/js/editor

Unfortunately I'm not able to figure out how to use with my own widgets, and the documentation is just about none.

Can we get some docs on this please?

**Comment by LucasMariniFalbo** [comment]
I've been struggling with the same thing.

**Reply by iamntz** [reply]
@LucasMariniFalbo @nicholaszein replied (on Discord) is this matter:

> There isn't a official API available for this type of widget published yet. And it won't be available until the container has completely replaced sections, or nested elements are merged to core. 

> These elements keep evolving, and there are plenty of changes being made under the hood (especially performance-wise). So publishing an API now could potentially break custom widgets in the future that might use it now. For the tie being it's best not to attempt to reproduce them.

So, unfortunately the answer is _not yet_ :)
